### PR TITLE
update redis crate to 0.25

### PR DESCRIPTION
**Pull request by rajatjindal**
this supersedes #75

### CODE REVIEW

File: tests/src/integration_test.rs [hunk 1]
```diff
@@ -57,7 +57,7 @@ mod test {
         port_forward_redis(forward_port, redis_port).await?;
 
         let client = redis::Client::open(format!("redis://localhost:{}", forward_port))?;
-        let mut con = client.get_async_connection().await?;
+        let mut con = client.get_multiplexed_async_connection().await?;
```
Comment (devigned) [inline tests/src/integration_test.rs:60]: Deprecated: aio::Connection is deprecated. Use client::get_multiplexed_async_connection instead.

per: https://docs.rs/redis/latest/src/redis/client.rs.html#77

**Review by devigned — APPROVED** [review]
lgtm

**Comment by devigned** [comment]
@rajatjindal, it looks like the commit was not signed. Would you amend the commit and sign it?

**Review by Mossaka — APPROVED** [review]
Thanks! Will merge it once the signature is amended to the commit.

**Comment by rajatjindal** [comment]
👍  signed the commit now (pushed it from a new dev box).

**Pull request merged**